### PR TITLE
support zooming out for ol.interaction.DragZoom

### PR DESCRIPTION
**Pull request by drnextgis**

### CODE REVIEW

**Comment by ahocevar** [comment]
What is the intended use? As a second DragZoom interaction with a different condition? Will two DragZoom interactions on the same map work? Maybe an example would clarify things a bit.

File: src/ol/interaction/dragzoominteraction.js [hunk 1]
```diff
@@ -66,6 +80,9 @@ ol.interaction.DragZoom.prototype.onBoxEnd = function() {
   var currentCenter = view.getCenter();
   goog.asserts.assert(currentCenter !== undefined, 'center should be defined');
 
+  var resolution = view.constrainResolution(
```
Comment (ahocevar) [inline src/ol/interaction/dragzoominteraction.js]: Was there a reason for moving this block down?

Comment (drnextgis) [inline src/ol/interaction/dragzoominteraction.js]: Because `extent` values is changed within `if (this.out_)` block.

Comment (ahocevar) [inline src/ol/interaction/dragzoominteraction.js]: I meant why you moved it down to below the assertions.

Comment (drnextgis) [inline src/ol/interaction/dragzoominteraction.js]: This is not intentionally, I'll move it back.

**Comment by drnextgis** [comment]
Yes, it is intended to use as second DragZoom interaction with a different condition. Here is example: http://jsfiddle.net/_DR_/gxee94o6/

**Comment by ahocevar** [comment]
Ok cool. Can you please add a unit test and submit a [CLA](https://github.com/openlayers/cla). Thanks!

**Comment by drnextgis** [comment]
I've already submitted [CLA](https://github.com/openlayers/cla/blob/master/signed-agreements.md) some years ago: `Rykov, Denis Rykov (@dr)`

**Comment by ahocevar** [comment]
Thanks @drnextgis. Are you still using the @dr handle? If not, I'll update the CLA list.

**Comment by drnextgis** [comment]
Yes update it please. I've added unit test and perform PR rebasing.

**Comment by ahocevar** [comment]
Thanks @drnextgis.

**Pull request merged**